File: cash/cash.c
#include <cs50.h>
#include <stdio.h>

int calc_quaters(int cents);

int calc_dime(int cents);

int calc_nickel(int cents);

int calc_penny(int cents);

int main(void)
{
    int cents;
    do
    {
        cents = get_int("owed: ");
    }
    while(cents < 1);

    int quaters = calc_quaters(cents);

    cents = cents - (quaters * 25);

    int dime = calc_dime(cents);

    cents = cents - (dime * 10);

    int nickel = calc_nickel(cents);

    cents = cents - (nickel * 5);

    int penny = calc_penny(cents);

    cents = cents - (penny * 1);

    printf("%i\n",quaters+dime+nickel+penny);
}

int calc_quaters(int cents)
{
    int quaters = 0;
    while(cents >= 25)
    {
        quaters++;
        cents = cents - 25;
    }
    return quaters;
}

int calc_dime(int cents)
{

}

int calc_nickel(int cents)
{

}

int calc_penny(int cents)
{

}

File: sentimental-cash/cash.py
from cs50 import get_float

while True:
    change = get_float('change: ')
    if change > 0:
        break
change = change * 100

quaters = 0
while True:
    if change >= 25:
        quaters += 1
        change -= 25
    else:
        break

dime = 0
while True:
    if change >= 10:
        dime += 1
        change -= 10
    else:
        break

nickel = 0
while True:
    if change >= 5:
        nickel += 1
        change -= 5
    else:
        break

penny = 0
while True:
    if change >= 1:
        penny += 1
        change -= 1
    else:
        break

total = quaters + dime + nickel + penny

print(f'{total}')
